platforms.sh

```diff
@@ -292,15 +292,15 @@ init_desktop()
 {
   local THIS="platforms.sh>env_dispatch()>init_desktop()"
   scenarioMessage "$THIS: Setting platforms-specific parameters."
-  HPCENV=jason-desktop.seahorsecoastal.com
+  HPCENV=desktop.seahorsecoastal.com
   QUEUESYS=mpiexec
   QUEUENAME=mpiexec
   SERQUEUE=cmd
   QCHECKCMD="ps -aux | grep mpiexec "
   SUBMITSTRING="mpiexec "
   SCRATCH=/srv/asgs
-  SSHKEY=id_rsa_jason-desktop
-  ADCOPTIONS='compiler=gfortran MACHINENAME=jason-desktop'
+  SSHKEY=id_rsa_desktop
```

Comment:
do we use this? We should be removing `SSHKEY` altogether and just use `$HOME/.ssh/config`

Reply:
No, we don't use it and I was tempted to delete it, but was trying to keep the scope of changes as narrow as possible (already too wide as it is). I can log a separate issue to target the removal of these vestigial settings. Please let me know.

Reply:
I removed my name as part of our goal of redacting personally identifiable information.

Reply:
That's great. I think that you should just remove them altogether if that's okay with you.

Reply:
Up to you, I will mark this complete but might add a commit to finish taking it all out...or maybe make a new issue?